### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gagdiez**

### CODE REVIEW

File: test/__snapshots__/make.test.ts.snap [hunk 1]
```diff
@@ -254,84 +254,84 @@ and then use the logged account to sign the transaction: \`--accountId <your-acc
...
-  \\"name\\": \\"hello_near\\",
-  \\"version\\": \\"1.0.0\\",
-  \\"license\\": \\"(MIT AND Apache-2.0)\\",
-  \\"type\\": \\"module\\",
-  \\"scripts\\": {
-    \\"build\\": \\"./build.sh\\",
-    \\"deploy\\": \\"./deploy.sh\\",
-    \\"test\\": \\"echo use integration-tests\\"
+  "name": "hello_near",
+  "version": "1.0.0",
```
Comment (frol) [inline test/__snapshots__/make.test.ts.snap:305]: I wonder if this actually works (quotes not being escaped feels wrong to me)

Comment (gagdiez) [reply test/__snapshots__/make.test.ts.snap:305]: those files are generated automatically, I do not think we should manually tinker with them

**Review by frol — APPROVED** [review]
I hope you know what you are doing 😄 

The change looks fine to me

File: test/__snapshots__/make.test.ts.snap (hunk 1 above)
Comment (frol) [reply test/__snapshots__/make.test.ts.snap:305]: I agree, but it looks odd to me, so I just flagged it in case that would trigger some alarm in your head

**Pull request closed without merging**